Formula/viddy.rb

```diff
@@ -0,0 +1,18 @@
+class Viddy < Formula
+  desc "Modern watch command"
+  homepage "https://github.com/sachaos/viddy"
+  url "https://github.com/sachaos/viddy/archive/refs/tags/v0.3.0.tar.gz"
+  sha256 "46a7705b97d985ca3bc4ffbc47ab0bdfa14ec9f7907fee041014134b89983782"
+  license "MIT"
+  head "https://github.com/sachaos/viddy.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=v#{version}")
+  end
+
+  test do
+    system "#{bin}/viddy", "-h"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
@carlocab Thank you for your review!
Unfortunately, viddy is a TUI app and currently requires user input.
If a feature or option is added that does not require user input, I would like to write a test using it, but it is currently difficult.

Reply:
Can we simulate pty input using Ruby's pty module?

Reply:
@carlocab Thank you! I've added a test by using Ruby's pty module.